Applications/Utils/FileConverter/NetCdfConverter.cpp

```diff
@@ -31,7 +31,8 @@
 
 using namespace netCDF;
 
-const double no_data = -9999;
+const double no_data_output = -9999;
+double no_data_input = -9999;
```

Comment:
```suggestion
static constexpr double no_data_output = -9999;
static double no_data_input = -9999;
```
`no_data_input` could be local I think. As well as no_data_output. But at least both must have internal linkage.

Reply:
Both are used in different places. I'd prefer to keep them in the beginning of the file.  
I've set both ```static```, though.